src/style.css

```diff
@@ -0,0 +1,21 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: inline-block;
+  margin-right: 4px;
```

Comment:
what about the last stars? do they need `margin-right`?

Reply:
done